.circleci/config.yml

```diff
@@ -237,6 +237,7 @@ jobs:
 
       - run: |
           bazel test --config=ci -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
+          bazel test --config=ci --//android/player:runtime=hermes -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
```

Comment:
This, plus the one below ensure we re-run all relevant tests (that depend on `//android/player:runtime`) against the Hermes runtime. It'll return the cached result for all tests that don't use that config flag, directly or transitively.